modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/client/blockstream/BlockStreamStorage.kt

```diff
@@ -0,0 +1,16 @@
+package jp.co.soramitsu.iroha2.client.blockstream
+
+import jp.co.soramitsu.iroha2.generated.VersionedBlockMessage
+import kotlinx.coroutines.channels.Channel
+import java.util.UUID
+
+data class BlockStreamStorage(
```

Comment:
can't that be a part of `BlockStreamContext`?

Reply:
we can move 3 common fields in these classes to another class
but I don't think we can merge these classes

Reply:
I guess we can merge though 🤔 
At least moving 3 common fields will be better